### ISSUE TITLE
suggestions for containerizing challenges ?

### ISSUE THREAD

**Issue by crowell** [pwning/docs#9]
for bkp, i use debootstrap + chroot, but have in the past used docker and perhaps some use systemd-nspawn (for exploit tasks)

would sections on these be desirable?

**Comment by percontation** [comment]
Yes. "How do I docker my challenge?" or similar comes up from time to time and it would be nice to have an answer to refer to.
